lib/ansible/vars/manager.py

```diff
@@ -179,6 +179,14 @@ def get_vars(self, play=None, host=None, task=None, include_hostvars=True, inclu
             _hosts_all=_hosts_all,
         )
 
+        _vars_sources = dict()
```

Comment:
Note that we've moved to asking people to use literal notation for empty dict, list, etc a few years ago.

Reply:
```suggestion
        _vars_sources = {}
```

Reply:
Meaning `_vars_sources = {}`?

Reply:
Ah, I see you put that in a suggestion. That suggested change doesn't show up in the conversation view on the issue...